src/main/java/com/puppycrawl/tools/checkstyle/utils/TokenUtil.java

```diff
@@ -305,6 +325,18 @@ public static boolean isOfType(DetailAST ast, int... types) {
         return ast != null && isOfType(ast.getType(), types);
     }
 
+    /**
+     * Determines if the AST belongs to the given types.
+     *
+     * @param ast the AST node to check
+     * @param types the acceptable types
+     *
+     * @return true if type matches one of the given types.
+     */
+    public static boolean isOfType(DetailAST ast, List<Integer> types) {
+        return isOfType(ast.getType(), types);
```

Comment:
Please remove this method .
No need in this abstraction layer.

Reply:
Done